domain/host/host.go

```diff
@@ -188,13 +193,26 @@ func Build(ip string, rpcport string, poolid string, memory string, ipAddrs ...s
 	}
 	host, err := currentHost(ip, rpcPort, poolid)
 	if err != nil {
-		glog.V(2).Infof("currentHost failed: %s", err)
+		plog.WithError(err).WithFields(log.Fields{
+			"ip": ip,
+			"rpcPort": rpcPort,
+			"poolID": poolid,
+		}).Debug("currentHost failed")
 		return nil, err
 	}
-	glog.Infof("Building host %s (%s) with ipsAddrs: %v [%d]", host.ID, host.IPAddr, ipAddrs, len(ipAddrs))
+	plog.WithFields(log.Fields{
+		"hostID": host.ID,
+		"ipAddr": host.IPAddr,
+		"ipAddrs": ipAddrs,
+		"len(ipAddrs)": len(ipAddrs),
```

Comment:
This shouldn't be logged here, or it should be logged at debug level. Audience for Info-level is an operator, and should only be logged after positive work is completed (e.g., `log.WithFields(logrus.Fields{"ipaddress": hostIP}).Info("Created host")`). Also, `len(ipAddrs)` is not a field we would want to publish, probably. Field names should be descriptive in human terms.